src/App.css

```diff
@@ -1,3 +1,64 @@
 .App {
-  text-align: center;
+  width: 100%;
+  height: 100vh;
+}
+
+:root {
+  --containersOpacity: 0;
+}
+
+#mapContainer {
+  width: 100%;
+  height: 100vh;
+  z-index: 0;
```

Comment:
The default value is 0, no need

Reply:
Good to know.
Isnt it worth it to leave it like that as a 'better safe than sorry' situation?

Reply:
Same as above, just verify that it actually works on Chrome, there should not be surprises.

Reply:
Implemented.